src/bors/handlers/review.rs

```diff
@@ -158,6 +162,7 @@ pub(super) async fn command_undelegate(
         .upsert_pull_request(
```

Comment:
Hmm, I see now that these upsert calls are very repetitive, and we pass a bunch of arguments to them. I'll take a look if we could do them one level above.